scripts/deployment/upload_model_to_huggingface.py

```
@@ -0,0 +1,1220 @@
+        'validation_warnings': missing_files
+    }
+
+def update_deployment_config(repo_name: str, model_info: dict[str, any]):
+    """Update deployment configurations to use the new model."""
+    print("\n🔧 UPDATING DEPLOYMENT CONFIGURATIONS")
+    print("=" * 40)
+
+    # Update model_utils.py to use the new model
+    model_utils_path = "deployment/cloud-run/model_utils.py"
+
+    if os.path.exists(model_utils_path):
```

Ruff (0.12.2)

`os.path.exists()` should be replaced by `Path.exists()`

(PTH110)